File: src/main/java/de/hsa/bpmnandai/project/infrastructure/entity/BankLoanRequestEntity.java
package de.hsa.bpmnandai.project.infrastructure.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity(name = "bank_credit_approval")
public class BankLoanRequestEntity {

    @Id
    @Column(name = "id", nullable = false)
    private String id;

    @Column(name = "Gender")
    private String gender;

    @Column(name = "City_Category")
    private String cityCategory;

    @Column(name = "Employer_Category1")
    private String employerCategory;

    @Column(name = "Monthly_Income")
    private Double monthlyIncome;

    @Column(name = "Contacted")
    private String contacted;

    @Column(name = "Source_Category")
    private String sourceCategory;

    @Column(name = "Existing_EMI")
    private Double existingEmi;

    @Column(name = "Loan_Amount")
    private Integer loanAmount;

    @Column(name = "Loan_Period")
    private Integer loanPeriod;

    @Column(name = "Interest_Rate")
    private Double interestRate;

    @Column(name = "EMI")
    private Integer emi;

    @Column(name = "is_approved")
    private String isApproved;

    @Column(name = "prediction_is_approved")
    private String predictionIsApproved;

    public void setPredictionIsApproved(final String predictionIsApproved) {
        this.predictionIsApproved = predictionIsApproved;
    }
}

File: src/main/kotlin/de/emaarco/bpmnandai/project/infrastructure/entity/BankLoanRequestEntity.kt
package de.emaarco.bpmnandai.project.infrastructure.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

@Entity(name = "bank_credit_approval")
class BankLoanRequestEntity(

    @Id
    @Column(name = "id", nullable = false)
    var requestId: String,

    @Column(name = "Gender")
    var gender: String? = null,

    @Column(name = "City_Category")
    val cityCategory: String? = null,

    @Column(name = "Employer_Category1")
    val employerCategory: String? = null,

    @Column(name = "Monthly_Income")
    val monthlyIncome: Double? = null,

    @Column(name = "Contacted")
    val contacted: String? = null,

    @Column(name = "Source_Category")
    val sourceCategory: String? = null,

    @Column(name = "Existing_EMI")
    val existingEmi: Double? = null,

    @Column(name = "Loan_Amount")
    val loanAmount: Int? = null,

    @Column(name = "Loan_Period")
    val loanPeriod: Int? = null,

    @Column(name = "Interest_Rate")
    val interestRate: Double? = null,

    @Column(name = "EMI")
    val emi: Int? = null,

    @Column(name = "is_approved")
    val isApproved: String? = null,

    @Column(name = "prediction_is_approved")
    var predictionIsApproved: String? = null,

    @Column(name = "prediction_is_approved_2")
    var predictionIsApproved2: String? = null

)